src/test/java/sentinel/utils/SentinelListTest.java

```diff
@@ -32,13 +32,13 @@ public void testSizeOneWhenSizeIsOne() {
         singleTaskList.add(new ToDo("Single Task"));
 
         // Act & Assert
-        assertTrue(singleTaskList.sizeOne());
+        assertTrue(singleTaskList.isSizeOne());
     }
 
     @Test
     public void testSizeOneWhenSizeIsNotOne() {
         // Act & Assert
-        assertFalse(sentinelList.sizeOne());
+        assertFalse(sentinelList.isSizeOne());
```

Comment:
Good work adhering to the coding standard here and other areas where you made this change 👍